plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -15,7 +15,8 @@ class AdviceLoader(global: Global, url: String, localStoreDir: File)(implicit ec
   private val OneDayMillis = 1000L * 60 * 60 * 24
 
   private val localStore = new File(localStoreDir, "clippy.json.gz")
-  private val projectAdviceFile = new File(".clippy.json")
+  private val projectRoot = sys.props.get("user.dir").getOrElse(".")
```

Comment:
As for testing locally, I was unable to actually publish and test locally, kept getting issue that I could not download from scala-clippy.org, figured maybe it was an authentication issue since I don't have the correct creds to publish.  Never dug deeper into it but all unit tests passed.